corehq/apps/userreports/models.py

```diff
@@ -97,8 +97,14 @@ class SQLPartition(DocumentSchema):
     constraint = StringProperty()
 
 
+class CitusConfig(DocumentSchema):
+    distribution_type = StringProperty(choices=['reference', 'hash'])
+    distribution_column = StringProperty()
+
+
 class SQLSettings(DocumentSchema):
     partition_config = SchemaListProperty(SQLPartition)
```

Comment:
What are your thoughts on removing this after the citus work is done? As of now, its only ever been used for ICDS, and we'd want it to be changed to using native partitioning on pg > 10 so it seems like it'd make sense to kill it?